python_benchmarks/global_params.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import torch
+
+# Correctness Tests tolerance
+RTOL = 1e-3
+ATOL = 1e-3
```

Comment:
We probably should just mark these literally in each benchmark since the level should depend on the operations applied in the fusion.

Reply:
Yes, I was thinking of using the constants here: https://github.com/NVIDIA/Fuser/blob/6d4abc04dc4939b10e4076d4f70101a818084d26/test/validator.h#L25C3-L25C3 since even within the same benchmark, the threshold needs to be varied based on size.

Reply:
We should make `testValidate` usable from Python as well. Doesn't need to be in this PR.